composer.json

```diff
@@ -24,7 +24,9 @@
         "doctrine/annotations": "^1.0",
         "doctrine/cache": "^1.0",
         "doctrine/collections": "^1.0",
-        "doctrine/event-manager": "^1.0"
+        "doctrine/event-manager": "^1.0",
+        "psr/cache": "^1.0",
+        "symfony/cache": "^4.4|^5.0"
```

Comment:
This is used to provide the forward/compatibility layer. Once we drop that in 3.0, this becomes a dev dependency.

Reply:
`symfony/cache` has a dev-dependency on `doctrine/cache`, so this means a circular dependency (not for us, but for them when they are in dev mode). I think this might be OK though since it's not mean to last, and since symfony seems to be resorting to `COMPOSER_ROOT_VERSION` in its CI already: https://github.com/symfony/symfony/search?q=COMPOSER_ROOT_VERSION

Still, we might want to check if we don't break their pipeline after releasing this (or merging it? I think they have builds that use untagged versions).

Reply:
How is there a circular dependency? This is doctrine/persistence, not doctrine/cache.

Reply:
> How is there a circular dependency?

`symfony/symfony` depends on `doctrine/persistence` and replaces `symfony/cache`. Anyway, we should be good here. 😃

Reply:
Right, nevermind my comment, not sure what I meant but I must have been confused.